src/AbstractAlgebra.jl

```diff
@@ -19,7 +19,7 @@ import GroupsCore: gens, ngens, order, mul!, istrivial
 
 # A list of all symbols external packages should not import from AbstractAlgebra
 const import_exclude = [:import_exclude, :QQ, :ZZ,
-                  :RealField, :number_field,
+                  :RealField, :number_field, :NumberField, :GF,
```

Comment:
Actually, why is this list defined here and not in Nemo? What are the other "external packages" that might be using this list, and who are we to decide what they should or should not import from AA ?!

For Nemo, there are clear reasons in most of these cases (either it defines its own version of those; or in the case of `Set` and `Module`, there are clashes). It is less clear to me for others, e.g. why is `Group` in this list?

Reply:
Pinging @thofma and @fieker who may know the reason

Reply:
For some of them, it was to not make some changes breaking (https://github.com/Nemocas/AbstractAlgebra.jl/pull/205#issuecomment-444933783).

The `div`, `rem` , etc are there, because they differ from the respective Base version, so doing an "import all"/"export all" downstream would be bad.